llm/app/core/redis_manager.py

```
@@ -46,29 +50,92 @@ async def _initialize_streams(self):
+                
+                # 모든 consumer의 pending 메시지 조회
+                pending_messages = await self.redis_client.xpending_range(
+                    self.settings.redis_stream_data_results,
+                    self.settings.redis_consumer_group,
+                    min="-",
+                    max="+",
+                    count=100
+                )
+                
+                # 오래된 pending 메시지들을 ACK 처리 (10분 이상 된 것들)
+                current_time = asyncio.get_event_loop().time() * 1000  # milliseconds
```

Ruff (0.12.2)

Local variable `current_time` is assigned to but never used

Remove assignment to unused variable `current_time`

(F841)